api/base/filters.py

```diff
@@ -61,6 +66,83 @@ def filter_queryset(self, request, queryset, view):
         return queryset
 
 
+class NewDjangoFilterMixin(django_filters.FilterSet):
+
+    QUERY_PATTERN = re.compile(r'^filter\[(?P<fields>((?:,*\s*\w+)*))\](\[(?P<op>\w+)\])?$')
+    FILTER_FIELDS = re.compile(r'(?:,*\s*(\w+)+)')
+
+    or_fields = []
+
+    def __init__(self, data=None, *args, **kwargs):
+        if data:
+            new_qd = QueryDict('', mutable=True)
+            for key, value in data.iteritems():
+                match = self.QUERY_PATTERN.match(key)
+                if match:
+                    match_dict = match.groupdict()
+                    fields = match_dict['fields']
+                    field_names = re.findall(self.FILTER_FIELDS, fields.strip())
+
+                    self.or_fields = []
+                    multiple_values = value.split(',')
+                    if len(multiple_values) > 1 or len(field_names) > 1:
```

Comment:
Cool cool, looks like it might not be supported OOB. [MultipleChoiceFilter](https://github.com/carltongibson/django-filter/blob/7e949076a5dec69811d081fd78838999c8abf374/django_filters/filters.py#L229) does something similar to what we want, except without a constrained set of "choices".  So maybe we can at least steal some code from that class.